manifests/container-storage-interface/oci-csi-controller-driver.yaml

```diff
@@ -74,6 +90,23 @@ spec:
               readOnly: true
             - mountPath: /var/run/shared-tmpfs
               name: shared-tmpfs
+        - name: oci-csi-fss-controller-driver
```

Comment:
since we start the FSS controller as a go routine, no need to add it as a container here.